src/Plugin.php

```diff
@@ -117,23 +114,36 @@ private function hook() {
 		add_filter( 'action_scheduler_store_class', [ $this, 'set_store_class' ], 10, 1 );
 		add_filter( 'action_scheduler_logger_class', [ $this, 'set_logger_class' ], 10, 1 );
 		add_action( 'plugins_loaded', [ $this, 'register_cli_command' ], 10, 0 );
-		add_action( 'plugins_loaded', [ $this, 'hook_scheduled_migration' ], 1000, 0 );
+		add_action( 'init', [ $this, 'maybe_hook_migration' ] );
```

Comment:
RE: the use of `init` here and my [previous comment](https://github.com/Prospress/action-scheduler-custom-tables/pull/17#discussion_r179299879) I have a guess why I went with `plugins_loaded` now - `wp_cron()` runs on `'init'` with priority 10 also. So I _may_ have been trying to make sure `maybe_hook_migration()` ran before `wp_cron`, and therefore, that scheduled migrations also happened _on the first WP Cron request_, if that was the first request after installing the plugin. If that was the reasoning, it's unnecessary as it would save a few minutes at most in most cases, if anything.